projects/ui/src/lib/components/po-breadcrumb/samples/sample-po-breadcrumb-labs/sample-po-breadcrumb-labs.component.html

```diff
@@ -41,7 +41,7 @@
       name="favoriteService"
```

Comment:
Desabilitar os campos "Favorite Service" e "Params Service", caso breadcrumbItems for invalido.

``` 
 <po-input
      class="po-md-6"
      name="favoriteService"
     [(ngModel)]="favoriteService"
      p-clean
      p-help="Ex.: https://thf.totvs.com.br/sample/api/favorite"
      p-help="Ex.: https://po-sample-api.herokuapp.com/v1/favorite"
      p-label="Favorite service"
      [p-disabled]="!breadcrumbItems.length">
    </po-input>
    <po-input
      class="po-md-6"
      name="paramsService"
      [(ngModel)]="paramsService"
      p-clean
      p-help="Ex.: { id: 14, user: 'dev.po' }"
      p-label="Params service"
      [p-disabled]="!breadcrumbItems.length"
    >
    </po-input>
    ```